### ISSUE TITLE
Do we really need to save total_loss and total_gain per strategy?

### ISSUE THREAD

**Issue by jmonteer** [jmonteer/yearn-vaults-v3#66]
During a process_report call function, we add the gain or loss recorded into the StrategyParams struct. 

https://github.com/jmonteer/yearn-vaults-v3/blob/2095b9c6931c854b57c746ee3d1868dfdcbf5bdc/contracts/VaultV3.vy#L620
https://github.com/jmonteer/yearn-vaults-v3/blob/2095b9c6931c854b57c746ee3d1868dfdcbf5bdc/contracts/VaultV3.vy#L597

While reviewing this function, I realised it's just for logging purposes as it is not used elsewhere. 

Should we remove it?

**Comment by banteg** [comment]
seems like a comfy shortcut, but you are right, it's trivial to compute the running sum of those from logs

**Comment by storming0x** [comment]
Agree, easier to compute off chain, and in practice this has never been used other than in an outstanding event.

are historic or onchain APY on scope for v3? i know we discussed for a while a feature like an APY oracle

**Comment by jmonteer** [comment]
not for core code but it would be great if the accountant module proposed in #68 could take care of on-chain APY, as part of periphery

do you have a specific usecase?

**Comment by storming0x** [comment]
> do you have a specific usecase?

Nothing concrete from users or integrators, but i think APY oracle opens up interesting possibilities for downstream products, but yeah, not core and probably nice to have
